packages/contracts/contracts/L1/OptimismPortal.sol

```diff
@@ -35,6 +119,132 @@ contract OptimismPortal is DepositFeed, WithdrawalsRelay {
      * depositTransaction() function directly.
      */
     receive() external payable {
-        depositTransaction(msg.sender, msg.value, 30000, false, bytes(""));
+        depositTransaction(msg.sender, msg.value, 100000, false, bytes(""));
+    }
+
+    /**
+     * @notice Accepts deposits of ETH and data, and emits a TransactionDeposited event for use in
+     * deriving deposit transactions.
+     * @param _to The L2 destination address.
+     * @param _value The ETH value to send in the deposit transaction.
+     * @param _gasLimit The L2 gasLimit.
+     * @param _isCreation Whether or not the transaction should be contract creation.
+     * @param _data The input data.
+     */
+    function depositTransaction(
+        address _to,
+        uint256 _value,
+        uint64 _gasLimit,
+        bool _isCreation,
+        bytes memory _data
+    ) public payable {
+        // Differentiate between sending to address(0)
+        // and creating a contract
+        if (_isCreation && _to != address(0)) {
+            revert NonZeroCreationTarget();
+        }
+
+        address from = msg.sender;
+        // Transform the from-address to its alias if the caller is a contract.
+        if (msg.sender != tx.origin) {
+            from = AddressAliasHelper.applyL1ToL2Alias(msg.sender);
+        }
+
+        emit TransactionDeposited(from, _to, msg.value, _value, _gasLimit, _isCreation, _data);
+    }
+
+    /**
+     * @notice Finalizes a withdrawal transaction.
+     * @param _nonce Nonce for the provided message.
+     * @param _sender Message sender address on L2.
+     * @param _target Target address on L1.
+     * @param _value ETH to send to the target.
+     * @param _gasLimit Gas to be forwarded to the target.
+     * @param _data Data to send to the target.
+     * @param _l2Timestamp L2 timestamp of the outputRoot.
+     * @param _outputRootProof Inclusion proof of the withdrawer contracts storage root.
+     * @param _withdrawalProof Inclusion proof for the given withdrawal in the withdrawer contract.
+     */
+    function finalizeWithdrawalTransaction(
+        uint256 _nonce,
+        address _sender,
+        address _target,
+        uint256 _value,
+        uint256 _gasLimit,
+        bytes calldata _data,
+        uint256 _l2Timestamp,
+        WithdrawalVerifier.OutputRootProof calldata _outputRootProof,
+        bytes calldata _withdrawalProof
+    ) external payable {
+        // Prevent reentrency
+        require(_target != address(this), "Cannot send message to self.");
+
+        // Get the output root.
+        L2OutputOracle.OutputProposal memory proposal = L2_ORACLE.getL2Output(_l2Timestamp);
+
+        // Ensure that enough time has passed since the proposal was submitted
+        // before allowing a withdrawal. A fault proof should be submitted
+        // before this check is allowed to pass.
+        require(
```

Comment:
Revert the `<`

Reply:
very important

Reply:
This is fixed in https://github.com/ethereum-optimism/optimistic-specs/pull/429